recordtbl_view.php

```diff
@@ -0,0 +1,432 @@
+			$thisWidth = ( strlen( $imgParsedData ) * $imgDataCharW ) + 5;
+			if ( $imgColumnWidths[$columnName] < $thisWidth )
+			{
+				$imgColumnWidths[$columnName] = $thisWidth;
+			}
+		}
+	}
+	// Calculate the image dimensions, create the image, and set the colours (black/white).
+	$imgWidth = array_sum( $imgColumnWidths ) + 1;
+	$imgHeight = $imgHeaderH + ( count( $resultTable ) * ( $imgDataH ) ) + 1;
+	$img = imagecreate( $imgWidth, $imgHeight );
+	imagecolorallocate( $img, 255, 255, 255 );
+	$imgBlack = imagecolorallocate( $img, 0, 0, 0 );
+	// Draw the column headers.
+	$posW = 0;
+	$posH = 0;
+	foreach ( $listColumns as $columnName )
+	{
+		$thisWidth = $imgColumnWidths[$columnName];
+		imagerectangle( $img, $posW, $posH, $posW + $thisWidth, $posH + $imgHeaderH, $imgBlack );
+		imagestring( $img, $imgHeaderFont, $posW + 2, $posH + 1, $columnName, $imgBlack );
+		$posW += $thisWidth;
+	}
+	// Draw each row of data.
+	$posW = 0;
+	$posH += $imgHeaderH;
+	foreach ( $resultTable as $resultRow )
+	{
+		foreach ( $listColumns as $columnName )
+		{
+			$imgParsedData = isset( $resultRow[$columnName] )
+			                    ? $module->escapeHTML( $resultRow[$columnName] ) : '';
+			$thisWidth = $imgColumnWidths[$columnName];
+			imagerectangle( $img, $posW, $posH, $posW + $thisWidth, $posH + $imgDataH, $imgBlack );
+			imagestring( $img, $imgDataFont, $posW + 2, $posH + 1, $imgParsedData, $imgBlack );
+			$posW += $thisWidth;
+		}
+		$posW = 0;
+		$posH += $imgDataH;
+	}
+	// Output the image as a PNG and exit.
+	imagepng( $img );
+	exit;
+}
+
+
+
+// Display the project header and report navigation links.
+
+require_once APP_PATH_DOCROOT . 'ProjectGeneral/header.php';
+$module->outputViewReportHeader( $reportConfig['label'], 'recordtbl', true );
+
+// Initialise the row counter.
+$rowCount = 0;
+
+
+// If a description is provided, output it here.
+if ( isset( $reportData['desc'] ) && $reportData['desc'] != '' )
+{
+?>
+<p class="mod-advrep-description"><?php
+	echo $module->parseDescription( $reportData['desc'] ); ?></p>
+<?php
+}
+
+
+?>
+<table id="mod-advrep-table" class="mod-advrep-datatable dataTable">
+ <thead>
+  <tr>
+<?php
+if ( count( $resultTable ) > 0 )
+{
+	foreach ( $listColumns as $fieldName )
+	{
+?>
+   <th class="sorting"><?php echo $module->escapeHTML( $fieldName ); ?></th>
+<?php
+	}
+}
+?>
+  </tr>
+ </thead>
+ <tbody>
+<?php
+foreach ( $resultTable as $resultRow )
+{
+	$rowCount++;
+?>
+  <tr>
+<?php
+	foreach ( $listColumns as $fieldName )
+	{
+?>
+   <td><?php echo $module->escapeHTML( $resultRow[ $fieldName ] ); ?></td>
+<?php
+	}
+	if ( $rowCount == 0 )
+	{
+?>
+  <tr><td>No rows returned</td></tr>
+<?php
+	}
+?>
+  </tr>
+<?php
+}
+?>
+ </tbody>
+</table>
+<?php
+
+if ( $rowCount > 0 )
+{
+?>
+<p>Total rows returned: <span id="filtercount"></span><?php echo $rowCount; ?></p>
+<?php
+}
+
+
+$module->outputViewReportJS();
+
+
+// Display the project footer
+require_once APP_PATH_DOCROOT . 'ProjectGeneral/footer.php';
```

Comment:
I think this must have been a permissions issue. The record table applies the data export permission for the user to the output. If a form is created and the user rights not updated it might have defaulted to 'no access'.